AttackItem.h

```diff
@@ -1 +1,9 @@
-#include "Item.h"
\ No newline at end of file
+#pragma once
+#include "Item.h"
+
+struct AttackItem : Item
+{
+    AttackItem() : Item("attack item", 11) { }
```

Comment:
Figure out the correct Boost amount based on the instructions in the other source files.
- [ ] revise